SA3/qa_web/search_indexes.py

```diff
@@ -0,0 +1,21 @@
+__author__ = 'jet'
+# import datetime  # RuntimeWarning: naive datetime is not the best time marker in django
+from django.utils import timezone
+from haystack import indexes
+from qa_web.models import Questions
+
+# If it's the first time you try to use search, please execute: python manage.py rebuild_index
+# The search index is configured to be automatically updated once a model instance is saved or deleted that
+# has an associated SearchIndex.
+
+class QuestionsIndex(indexes.SearchIndex, indexes.Indexable):
```

Comment:
This class is also not tested. If it is possible/necessary to test it, then we should add a new test file for it (such as `test_search_indexes.py`). Else if it cannot be tested for some reason, then its path should be added to the `[run] omit` property of `.coveragerc`.

Reply:
You're right. I will try to test it